src/core/services/gesture/gesture.js

```diff
@@ -643,7 +643,9 @@ function attachToDocument( $mdGesture, $$MdGestureHandler ) {
     updatePointerState(ev, pointer);
     pointer.endTime = +Date.now();
 
-    runHandlers('end', ev);
+    if (ev.type !== 'pointercancel') {
```

Comment:
Regarding to issues I mentioned in commit message, In chromium/chrome engine and webview in android there is a problem with angular material implementation of gestures. The problem is when we have swipable tabs with scrollable content dragging page to scroll fires pointercancel and causes to swipe to the other tab. The problem also can occure where ever we use drag and swipe together. So I excluded pointercancel from events to be handled by handler functions. If you want I can provide codepen to test, but the discussion on #10145 describes the issue well.